modules/sublimeBidAdapter.js

```diff
@@ -140,28 +150,23 @@ export const spec = {
         referrer: '',
         ad: serverResponse.body.ad || '',
       };
-      if (bidResponse.cpm) {
-        bidResponses.push(bidResponse);
-        top.sublime.analytics.fire('bid', {
-          qs: {
-            z: SUBLIME_ZONE
-          }
-        });
+
+      if (response.timeout) {
+        // Debug timeout from the long polling server
+        sendAntennaPixel('dlptimeout');
+      } else if (bidResponse.ad.match(regexNoAd)) {
+        // Debug LP response no ad (a=0 in the notify)
+        sendAntennaPixel('dlpnoad');
+      } else if (bidResponse.ad === '') {
+        // Debug no ad in the interpret response, what happenned ?
+        sendAntennaPixel('drespnoad');
       } else {
-        // debug pixel no cpm
-        top.sublime.analytics.fire('dirnocpm', {
-          qs: {
-            z: SUBLIME_ZONE
-          }
-        });
+        sendAntennaPixel('bid');
+        bidResponses.push(bidResponse);
       }
     } else {
       // debug pixel no request
```

Comment:
// debug pixel no response

Reply:
I didn't renamed the pixel, so I let `request` for now